kube-eagle/templates/deployment.yaml

```diff
@@ -16,6 +16,9 @@ spec:
   template:
     metadata:
       annotations:
+{{- with .Values.podAnnotations }}
+{{- toYaml . | nindent 8 }}
```

Comment:
Since you are using `nindent` you could indent it correctly for readability.

Reply:
Done